core/src/test/java/kafka/admin/ConfigCommandIntegrationTest.java

```diff
@@ -256,6 +278,32 @@ public void testDynamicBrokerConfigUpdateUsingKraft() throws Exception {
         }
     }
 
+    @ClusterTest(types = {Type.KRAFT, Type.CO_KRAFT}, serverProperties = {
+        @ClusterConfigProperty(key = "group.coordinator.new.enable", value = "true"),
+        @ClusterConfigProperty(key = "group.coordinator.rebalance.protocols", value = "classic,consumer"),
```

Comment:
Can we use the constants instead of the raw strings? It should work from here.

Reply:
I tried to do it, but failed.
![image](https://github.com/user-attachments/assets/cfab879d-7473-4e10-9a6a-408fe530b23b)

Reply:
Done.